### ISSUE TITLE
font metric data not found or bad

### ISSUE THREAD

**Issue by rpspringuel** [gregorio-project/gregorio#1458]
I've stripped things back to a bare TeX Live 2018 installation (no additional versions of Gregorio on top, so running 5.1.1) on my Mac and I keep getting the following error when compiling Gregorio documents:

```
/usr/local/texlive/2018/texmf-dist/tex/luatex/gregoriotex/gregoriotex-main.tex:
993: Font \gre@font@variant@B=name:greciliae-hollow at 25.9399pt not loadable: 
metric data not found or bad.
```

I've tried erasing the font cache, but am still running into the error.  Does anyone else have any ideas as to what might be going on here and how I can fix it?

**Comment by rpspringuel** [comment]
Fixed, though I forget exactly how at this point.

**Comment by rpspringuel** [comment]
Okay, I'm now working with TeX Live 2019, and this problem is recurring.  I'm not sure what causes the problem is, as it was working just fine a week ago (when I did all the work on the ligatures).

After some playing around I've found that the solution is `luaotfload-tool --update --force`.  This forces a complete rebuild of the font database and solves the problem.  Obviously this isn't an issue with our package, but I'm documenting how I fixed it here so that I can find the solution again should the problem recur.

**Comment by rpspringuel** [comment]
Okay, while the issue is solved for normal documents, I'm still running into it when attempting to build the documentation through `make` (presumably because the system is trying to use the just built fonts, rather than the installed ones).  I can get around this by building the documentation using the installed packages (and then `make` seeing they are already present will happily go on to the next step) but there's probably a better way (as this would break in those cases when the new fonts are needed to create the updated documentation).

So, how to do the equivalent of `luaotfload-tool --update --force` for the newly built fonts in the `fonts` directory during the build process?

**Comment by henryso** [comment]
It's kind of ugly, but maybe for "production" builds, include that command in the Makefile (e.g., add some flag that is used during "production" builds)?

**Comment by rpspringuel** [comment]
Well, the command by itself won't help as it only fixes the problem for installed fonts.  For fonts found by using the `TTFONTS` variable, updating the fonts database doesn't help.

There is a `--local` option for `luaotfload-tool`.  It looks specifically in `$PWD`, but perhaps that can be leveraged to solve this problem.  I'll have to explore that after I get back from vacation.

**Comment by rpspringuel** [comment]
Fixed on develop by 1897297.
